core/src/core/transaction.rs

```diff
@@ -234,6 +234,19 @@ pub struct Transaction {
 	pub offset: BlindingFactor,
 }
 
+/// PartialEq
```

Comment:
Can we just `derive` PartialEq on Transaction (we're just comparing all the fields)?

Reply:
That's what I did at the beginning and for some reasons the comparison were not the same.

Reply:
I don't think you need the `if` though...